src/dashboard/src/components/administration/forms.py

```diff
@@ -294,6 +294,7 @@ def __init__(self, *args, **kwargs):
                     for item in replace_dicts:
                         choices.append((item.pk, item.description))
                 elif ftype == 'storage_service':
+                    choices.append(('/api/v2/location/default/{}/'.format(field['purpose']), _('Default location')))
```

Comment:
This is here so the user can choose the "default" from the web interface (processing configuration page).